EosGround/radioDriver.py

```diff
@@ -24,12 +24,12 @@
 cursor = conn.cursor()  # creates cursor
 
 # sets up digi
-PORT = "COM12"
+PORT = "COM15"
```

Comment:
It looks like David hijacked your branch.  If he wants to work on the command interface, he can make his own branch like he knows he's supposed to.

You can either address these comments, or you could just run this command:
`git revert 8fc433a6716a718a933095d2cf5f3159ff0b8624`
I'll leave it up to you....